client/web/package.json

```diff
@@ -59,15 +59,16 @@
     "webpack-merge": "^5.7.3"
   },
   "dependencies": {
-    "@genesislcap/foundation-comms": "^6.0.0",
-    "@genesislcap/foundation-entity-management": "^6.0.0",
-    "@genesislcap/foundation-header": "^6.0.0",
-    "@genesislcap/foundation-login": "^6.0.0",
-    "@genesislcap/foundation-ui": "^6.0.0",
-    "@genesislcap/foundation-utils": "^6.0.0",
-    "@genesislcap/foundation-zero": "^6.0.0",
-    "@genesislcap/foundation-zero-grid-pro": "^6.0.0",
-    "@genesislcap/g2plot-chart": "^6.0.0",
+    "@genesislcap/alpha-design-system": "10.6.0",
```

Comment:
Long-term the idea is that we'd generate a DS like alpha based on genx answers (so if you choose prefix `client`, it'd be `client-design-system` rather than always alpha - alpha is just an example). For this PR this is fine.